docs/examples/imageset-config-mirror-to-mirror-with-catalogs-from-disk.yaml

```diff
@@ -0,0 +1,41 @@
+# This configuration demonstrates how to include some locally built operator catalogs
+# to be mirrored to a partially disconnected cluster.
+
+# This feature might be interesting when needing to mirror internal operator catalogs
+# that are not pushed to registries, such as outcomes of a CI/CD pipeline for example.
+
+# These local catalogs need to be :
+# - File based catalogs (as opposed to SQLite)
+# - in OCI format
+
+# This ImageSet configuration can only be used to mirror to a registry that is reachable,
```

Comment:
Assuming this it is because the `oci` feature does not support the mirror to disk workflow. Perhaps that should be included, unless it is documented somewhere else.